front/app/containers/Admin/reporting/components/ReportBuilder/Editor/RenderNode/index.tsx

```diff
@@ -142,10 +150,12 @@ const RenderNode = ({ render }) => {
           : 'transparent'
       }
       isRoot={id === ROOT_NODE}
-      onMouseOver={() => {
+      onMouseOver={(e) => {
```

Comment:
Also unrelated- this way, only the borders of the element you are hovering over are shown, and not the borders of the parent element (such as the two column layout).